packages/siop-oid4vp/lib/types/VpJwtVerifier.ts

```diff
@@ -96,10 +96,10 @@ export const getRequestObjectJwtVerifier = async (
     if (jwt.payload.redirect_uri && jwt.payload.redirect_uri !== clientId) {
       throw new Error(SIOPErrors.INVALID_CLIENT_ID_MUST_MATCH_REDIRECT_URI)
     }
-    const parts = options.raw.split('.')
+    /*const parts = options.raw.split('.')  this can be signed and execution shouldn't even be here when alg = none 
```

Comment:
Why are we removing this, instead of not checking it when alg === none ?

Reply:
Execution never gets here if alg is none
line 83
```
  if (!clientIdScheme || jwt.header.alg === 'none') {
    return getJwtVerifierWithContext(jwt, { type })
  }
```